Products/zms/_confmanager.py

```diff
@@ -1047,7 +1047,7 @@ def exec_auto_update(self): return True
     ############################################################################
 
     def getWorkflowManager(self):
-      manager = [x for x in self.getDocumentElement().objectValues() if x.getId() == 'workflow_manager']
+      manager = [x for x in self.getDocumentElement().objectItems() if type(x)!=tuple and x.getId() == 'workflow_manager']

```

Comment:
if you already know that you expect an object with the id 'workflow_manager' then you should directly access it with getattr(self.getDocumentElement(),'workflow_manager',None)

Reply:
reverting/downgrading from PR https://github.com/zms-publishing/ZMS/pull/255 may result in an "OverflowError" (mktime argument out of range); this will be fixed here:
https://github.com/zms-publishing/ZMS/pull/256/commits/6f5e5519a93c0be3a4c2487de0b5dc5a5e1e34e2